src/NuGet.Services.AzureSearch/SearchService/SearchParametersBuilder.cs

```diff
@@ -93,7 +94,7 @@ public SearchParameters Autocomplete(AutocompleteRequest request, bool isDefault
         {
             var searchParameters = NewSearchParameters();
 
-            ApplySearchIndexFilter(searchParameters, request, isDefaultSearch);
+            ApplySearchIndexFilter(searchParameters, request, isDefaultSearch, packageType: null);
```

Comment:
Do we know that the .NET CLI folks don’t plan to use autocomplete endpoint? Maybe we should ask them? No need to block this PR on that though :)

Reply:
Hey, you're on vacation! Get outta here.

Good point, I asked them. Should be easy to add one way or another but if they don't expect to use it we can exclude it from the MVP.